### PR TITLE
Map Open key in Sign.proj, don't set signing properties when not signing

### PR DESCRIPTION
**Pull request by ericstj**

### CODE REVIEW

File: src/Microsoft.DotNet.Arcade.Sdk/tools/StrongName.targets [hunk 1]
```diff
@@ -26,6 +26,7 @@
 
   <!-- Binaries are delay or public-signed with one of these keys; later, the signing system will finish the strong-name signing. -->
   <Choose>
+    <When Condition="'$(SignAssembly)' == 'false'" />
```
Comment (joperezr) [inline src/Microsoft.DotNet.Arcade.Sdk/tools/StrongName.targets]: I guess that with this we won't have to clear this property here?: https://github.com/dotnet/corefx/blob/2dff4d0374d1425505e314a71772e06a10aee2d3/src/System.Runtime.Loader/tests/System.Runtime.Loader.Noop.Assembly/System.Runtime.Loader.Noop.Assembly.csproj#L5-L7

Comment (ericstj) [reply src/Microsoft.DotNet.Arcade.Sdk/tools/StrongName.targets]: Right, this is an altenative to that.

File: src/Microsoft.DotNet.Arcade.Sdk/tools/Sign.proj [hunk 2]
```diff
@@ -22,6 +22,7 @@
     <StrongNameSignInfo Include="SilverlightCert121" PublicKeyToken="7cec85d7bea7798e" CertificateName="Microsoft400" />
     <StrongNameSignInfo Include="StrongName" PublicKeyToken="b77a5c561934e089" CertificateName="Microsoft400" />
     <StrongNameSignInfo Include="StrongName" PublicKeyToken="b03f5f7f11d50a3a" CertificateName="Microsoft400" />
+    <StrongNameSignInfo Include="$(MSBuildThisFileDirectory)snk\Open.snk" PublicKeyToken="cc7b13ffcd2ddd51" CertificateName="Microsoft400" />
```
Comment (JohnTortugo) [inline src/Microsoft.DotNet.Arcade.Sdk/tools/Sign.proj]: I was planning to add this sometime before I merge this PR https://github.com/dotnet/arcade/pull/1342 :-)  Thanks for working on this.

File: src/Microsoft.DotNet.Arcade.Sdk/tools/StrongName.targets (hunk 1 above)
Comment (ericstj) [reply src/Microsoft.DotNet.Arcade.Sdk/tools/StrongName.targets]: My other change to undo the DelaySign change for the open key makes this unnecessary but I still feel like it is correct.  We shouldn't be setting a bunch of signing properties when not signing.

**Pull request merged**